experiments/rtp-hdrext/video-timing/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: video-timing
+permalink: /experiments/rtp-hdrext/video-timing/
+---
+
+The Video Timing extension is used to communicate a timing information on per-frame basis
+to receiver of rtp video stream. Contact <ilnik@google.com> for more info. It may be generalized 
+to audio frames as well in the future.
+
+Name: "Video Timing" ; "RTP Header Extension for Video timing"
+
+Formal name: <http://www.webrtc.org/experiments/rtp-hdrext/video-timing>
```

Comment:
Actually, it shouldn't be https. It's not a link but a name. Http is used in all such names, so it should be the same here.